elife/backups.sls

```diff
@@ -44,3 +44,7 @@ monitor-backup-logs:
         - require:
             - pkg: syslog-ng
 
+rotate-backup-logs:
+    file.managed:
+        - name: /etc/logrotate.d/ubr
+        - source: salt://elife/config/etc-logrotate.d-ubr
```

Comment:
do you need to trigger in someway `logrotate` to re-read this? I don't think so as it's not a daemon, and should read it when started by `cron`

Reply:
I don't think so. logrotate, like most shell things, will pick up on the contents of it's `.d/` directory